apps/services/user-profile/src/app/v2/userToken.controller.ts

```diff
@@ -0,0 +1,47 @@
+import { ApiSecurity, ApiTags } from '@nestjs/swagger'
+import { Controller, Get, UseGuards, Headers } from '@nestjs/common'
+
+import { IdsAuthGuard, Scopes, ScopesGuard } from '@island.is/auth-nest-tools'
+import { UserProfileScope } from '@island.is/auth/scopes'
+import { Audit } from '@island.is/nest/audit'
+import { Documentation } from '@island.is/nest/swagger'
+
+import { UserTokenService } from './userToken.service'
+import { UserDeviceTokenDto } from '../user-profile/dto/userDeviceToken.dto'
+
+const namespace = '@island.is/user-profile/v2/userTokens'
+
+@UseGuards(IdsAuthGuard, ScopesGuard)
+@Scopes(UserProfileScope.admin)
+@ApiTags('v2/userTokens')
```

Comment:
I would use the same tag `v2/users` so the codegen will generate this in the same api class.